Content.Server/Medical/HealingSystem.cs

```diff
@@ -52,7 +57,15 @@ private void OnDoAfter(EntityUid uid, DamageableComponent component, HealingDoAf
 
         // Heal some bloodloss damage.
         if (healing.BloodlossModifier != 0)
+        {
+            var isBleeding = uid.EnsureComponent<BloodstreamComponent>().BleedAmount > 0;
```

Comment:
you can just do `TryComp` here instead of doing this twice, and you can still check the bound variable since this is stateful